src/slider.test.events.multiple.ts

```diff
@@ -0,0 +1,896 @@
+import { assert, aTimeout, expect, fixture, html } from '@open-wc/testing';
+import sinon from 'sinon';
+import { sendKeys } from '@web/test-runner-commands';
+import Slider from './slider.js';
+
+it('dispatches an "input" event typing in the minimum input', async () => {
```

Comment:
Strictly speaking, _it_ (the component) doesn't type into the minimum input.

```suggestion
it('dispatches an "input" event when its minimum is changed', async () => {
```